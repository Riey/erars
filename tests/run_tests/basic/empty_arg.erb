@SYSTEM_TITLE
CALL FOO, , , ,
CALL FOO(, , 1)

@FOO(ARG, ARGS, ARG:1)
PRINTFORML ARG`{ARG}` ARGS`%ARGS%` ARG:1`{ARG:1}`
