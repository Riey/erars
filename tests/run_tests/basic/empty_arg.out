ARG`0` ARGS`` ARG:1`0`
ARG`0` ARGS`` ARG:1`1`
